pages/home.md

```diff
@@ -8,88 +8,162 @@ permalink: /
   <div class="hero-body">
     <div class="container">
       <h1 class="title has-text-centered is-size-3 is-size-2-tablet is-size-1-desktop">
-        The Web — <a href="{{site.baseUrl}}/take3" title="Why Take 3?">Take 3</a>
+        Solid
       </h1>
       <p class="subtitle has-text-centered is-size-4 is-size-3-tablet is-size-2-desktop">
-        Solid is a technology, like the Web, but a new level of standard which adds to the existing protocols to make it more powerful, particularly to empower individuals at home and at work.
+      Using Web standards to
+      let people control their data, and choose the
+      applications and services to use with it.
       </p>
+      <div class="buttons is-centered">
+        <!--
+        <a href="{{site.baseurl}}/about/">
+          <button class="button is-primary">About Solid</button>
+        </a>
+        -->
+        <a href="{{site.baseurl}}/users/">
+          <button class="button is-primary">Get a Pod</button>        
+        </a>
+        <a href="{{site.baseurl}}/developers/tutorials/getting-started">
+          <button class="button is-primary">Developer Tutorial</button>
+        </a>
```

Comment:
A couple of changes:

- Screen readers won't be able to make sense of buttons inside anchors, probably - `<button>`s removed.
- Increased the button sizes to fit the visual hierarchy.
- Only made a single button the primary call to action to clearly guide the eye.

```suggestion
        <a href="{{site.baseurl}}/users/" class="button is-primary is-large">
          Get a Pod
        </a>
        <a href="{{site.baseurl}}/developers/tutorials/getting-started" class="button is-info is-outlined is-large">
          Developer Tutorial
        </a>
```